packages/ui/css/base.css

```diff
@@ -9,6 +9,7 @@
 @import "./components/icon";
 @import "./components/form";
 @import "./components/error-message";
+@import "./components/RadioList.css";
 
```

Comment:
@tristanMatthias What do you think if we add
 
```
html {
  font-size: 62.5%;
}
```
here so we can have 1.6rem = 16px?

Reply:
Messes with tailwind a lot